gui/doc/input.csv

```diff
@@ -1,5 +1,6 @@
 ﻿Name,Type,Default,Doc
 *value,dynamic(any),None,The value represented by this control.
+password,bool,False,"Sets the input as a password (ie character are displayed as *)."
```

Comment:
-> "If True, the text is obscured: all input characters are displayed as an asterisk ('*')."

Note for Fred: if this is not dynamic, we cannot create any 'Show password' toggle button, right? Just sayin'

Reply:
I guess this would never need to be dynamic.
We could add an option to allow the user to reveal the password but it would be anoter property and the reveal would be local